### PR TITLE
Staff chat distinction

### PR DESCRIPTION
**Pull request by andrewknoll**
- Staff will now have the "Staff - " prefix, rather than "Owner - " in a way to make it easier for the players to identify them.
- Staff will now have a red box in the property sim list, instead of the Owner blue one.
This is to make it easier for players (especially newbies) to identify a Staff member.

### CODE REVIEW

File: TSOClient/tso.client/UI/Controls/UIVMPersonButton.cs [hunk 1]
```diff
@@ -94,7 +94,7 @@ public void UpdateAvatarState(VMTSOAvatarPermissions perm)
                 case VMTSOAvatarPermissions.Visitor: bgID = 0x25400000001; break; //personbuttontemplate_visitorlarge
                 case VMTSOAvatarPermissions.Roommate:
                 case VMTSOAvatarPermissions.BuildBuyRoommate: bgID = 0x25200000001; overlayID = 0xB7F00000001; break; //personbuttontemplate_roommatelarge, personbuttonoverlay_roommatelarge
-                case VMTSOAvatarPermissions.Admin: bgID = 0x25200000001; overlayID = 0x8B000000001; break;
+                case VMTSOAvatarPermissions.Admin: bgID = 0x25000000001; overlayID = 0x8B000000001; break;
```
Comment (andrewknoll) [inline TSOClient/tso.client/UI/Controls/UIVMPersonButton.cs:97]: Found this red file that works perfectly for this purpose

File: TSOClient/tso.client/UI/Controls/UIVMPersonButton.cs [hunk 2]
```diff
@@ -94,7 +94,7 @@ public void UpdateAvatarState(VMTSOAvatarPermissions perm)
                 case VMTSOAvatarPermissions.Visitor: bgID = 0x25400000001; break; //personbuttontemplate_visitorlarge
                 case VMTSOAvatarPermissions.Roommate:
                 case VMTSOAvatarPermissions.BuildBuyRoommate: bgID = 0x25200000001; overlayID = 0xB7F00000001; break; //personbuttontemplate_roommatelarge, personbuttonoverlay_roommatelarge
-                case VMTSOAvatarPermissions.Admin: bgID = 0x25200000001; overlayID = 0x8B000000001; break;
+                case VMTSOAvatarPermissions.Admin: bgID = 0x25000000001; overlayID = 0x8B000000001; break; // personbuttontemplate_playerlarge
```
Comment (andrewknoll) [inline TSOClient/tso.client/UI/Controls/UIVMPersonButton.cs:97]: This file is a red box that works perfectly for this purpose

File: TSOClient/tso.content/Content/UI/uitext/english.dir/_f114_communityingamestrings.cst [hunk 3]
```diff
@@ -17,4 +17,6 @@
 
 14 ^You have successfully invited this sim to be your roommate. You'll need to wait for their response before you see them on your Roommates list, though!^
 15 ^This sim is already a roommate in another property. You can only invite sims who do not live on any property!^
-16 ^A roommate invitation has already been sent to this sim, and is pending a response. You'll need to wait for them to respond before taking any further actions.^
\ No newline at end of file
+16 ^A roommate invitation has already been sent to this sim, and is pending a response. You'll need to wait for them to respond before taking any further actions.^
+
+17 ^Staff - ^
```
[the remark this answers is not in the document]
Comment (riperiperi) [reply TSOClient/tso.content/Content/UI/uitext/english.dir/_f114_communityingamestrings.cst:22]: Yeah, I guess there's no fso string location for generic ingame strings.

**Review by riperiperi — APPROVED** [review]
LGTM. Didn't realize that person button template was there, fits perfectly.

**Pull request merged**